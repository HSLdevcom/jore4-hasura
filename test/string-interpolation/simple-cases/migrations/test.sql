foo
xxx_bar_xxx
baz xxx_xyz_xxx
xxx_xyz_xxx sed
awk xxx_xyz_xxx sed
xxx_zyx_xxx
xxx_xyzxzyx_xxx
xxx_e_lep_hant_xxx
xxx__trunk__xxx
