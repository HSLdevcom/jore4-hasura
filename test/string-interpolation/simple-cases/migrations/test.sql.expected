foo
BaR
baz XyZ
XyZ sed
awk XyZ sed
ZyX
XyZxZyX
ElEpHaNt
TrUnK
